assets/js/src/form_editor/components/editor.jsx

```diff
@@ -107,9 +109,11 @@ export default () => {
                 settings={editorSettings}
                 useSubRegistry={false}
               >
-                <div className="interface-interface-skeleton__left-sidebar">
-                  {isInserterOpen && (<Inserter setIsInserterOpened={setIsInserterOpen} />)}
-                </div>
+                {(!isMobileViewport || isInserterOpen) && (
```

Comment:
It seems like the `!isMobileViewport` condition does not have any effect. If I understand properly, it should affect a default state: closed on mobile, open on desktop. Yet it doesn't work like that in my tests (maybe I'm not reproducing it correctly, e.g. with a clean install). It behaves as if this condition wasn't there, only the sidebar status determines if it will be open or closed on both screen types. But it's not very important since you can close the sidebar anyway. Can we remove this condition then?

Reply:
You are right. I have changed it.